rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -84,25 +84,19 @@ def testMolFormula(self):
       self.assertEqual(actual, expected)
 
   def testMQNDetails(self):
-    refFile = os.path.join(RDConfig.RDCodeDir, 'Chem', 'test_data', 'MQNs_regress.pkl')
-    refFile2 = os.path.join(RDConfig.RDCodeDir, 'Chem', 'test_data', 'MQNs_non_strict_regress.pkl')
+    refFile = os.path.join(os.path.dirname(__file__), 'test_data', 'MQNs_regress.pkl')
```

Comment:
Any particular reason you made these changes? They aren't wrong, but the pattern of using `RDConfig` is pretty common in the python tests

Reply:
oh - good question. I usually build via installation into a Schrödinger suite, because there are things that I want to test there. These test files don't get installed.

Reply:
I *think* (and you should verify this) that if you run cmake with `-DRDK_INSTALL_PYTHON_TESTS=ON`  the python tests and data will be installed.